src/main/java/com/echobox/api/linkedin/jsonmapper/DefaultJsonMapper.java

```diff
@@ -263,6 +265,11 @@ public DefaultJsonMapper(JsonMappingErrorHandler jsonMappingErrorHandler) {
         return null;
       }
 
+      if (URN.class.equals(type)) {
```

Comment:
I think this should be in `DefaultJsonMapper.toJavaType`?
Happy to discuss

Reply:
good point, in fact taking a look I have managed to simplify the conversion further.